setup.cfg

```diff
@@ -7,3 +7,5 @@ packages = find_namespace:
 [options.entry_points]
 distutils.commands =
 	custom-build = custom.build:Command
```

Comment:
An alternative to the `distutils.command` entry-point is to do everything in the `setuptools.finalize_distribution_options`, by updating `cmdclass`, for example:

```python
def install(dist):
    # This could be protected by a opt-in condition...
    dist.cmdclass.update(custom_build=Command)
    build = dist.get_command_obj("build")
    build.sub_commands = [*build.sub_commands, ("custom-build", None)]
```

Not that the number of entry-points matter... I just wanted to show that another alternative is also possible.